NuKeeper.BitBucket/BitbucketSettingsReader.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Linq;
+using NuKeeper.Abstractions;
+using NuKeeper.Abstractions.CollaborationPlatform;
+using NuKeeper.Abstractions.Configuration;
+
+namespace NuKeeper.BitBucket
+{
+    public class BitbucketSettingsReader : ISettingsReader
+    {
+        public Platform Platform => Platform.Bitbucket;
+
+        private static string Username { get; set; }
+        
+        public bool CanRead(Uri repositoryUri)
+        {
+            if (repositoryUri == null || !repositoryUri.ToString().ToUpperInvariant().Contains("bitbucket.org".ToUpperInvariant()))
+            {
```

Comment:
This can now be `repositoryUri.ToString().ContainsOrdinal("bitbucket.org")`

There's a new helper method on master